### PR TITLE
✨ Add Vpon Analytics to AMP analytics vendors #21769

### PR DESCRIPTION
**Pull request by vpon-sdk**
✨  Add Vpon Analytics to AMP analytics vendors

### CODE REVIEW

**Comment by vpon-sdk** [comment]
@torch2424 @sebastianbenz could you please review this PR as well?
Thanks!

**Comment by torch2424** [comment]
Hello! Sorry for not catching this. The team is currently helping out at AMP Conf Tokyo. We can get to this sometime next week. Thank you for your patience! 😄

File: extensions/amp-analytics/0.1/vendors.js [hunk 1]
```diff
@@ -99,6 +99,8 @@ import {BG_CONFIG} from './vendors/bg';
 import {UPSCORE_CONFIG} from './vendors/upscore';
 import {REPPUBLIKA_CONFIG} from './vendors/reppublika';
 import {NAVEGG_CONFIG} from './vendors/navegg';
+import {VPONANALYTICS_CONFIG} from './vendors/vponanalytics';
+
```
[the remark this answers is not in the document]
Comment (vpon-sdk) [reply extensions/amp-analytics/0.1/vendors.js]: Done Fixed

File: extensions/amp-analytics/0.1/vendors/vponanalytics.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (vpon-sdk) [reply extensions/amp-analytics/0.1/vendors/vponanalytics.js]: Done Fixed

File: extensions/amp-analytics/0.1/vendors.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (vpon-sdk) [reply extensions/amp-analytics/0.1/vendors.js]: @torch2424 Thank for looking into it, could you please take a look why the test is failed now?

**Comment by torch2424** [comment]
Also, one last thing @vpon-sdk 

We would appreciate it if you could add yourself to our supported vendors page on our [amp.dev](https://amp.dev/) documentation repo 😄 

The file would be at: https://github.com/ampproject/docs/blob/master/content/docs/analytics/analytics-vendors.md

Thank you!

**Comment by vpon-sdk** [comment]
> Also, one last thing @vpon-sdk
> 
> We would appreciate it if you could add yourself to our supported vendors page on our [amp.dev](https://amp.dev/) documentation repo 😄
> 
> The file would be at: https://github.com/ampproject/docs/blob/master/content/docs/analytics/analytics-vendors.md
> 
> Thank you!

Thank you very much!
File has been added
Https://github.com/ampproject/docs/pull/1976

**Pull request merged**